charts/uws-api-server/templates/ingress.yaml

```diff
@@ -2,8 +2,7 @@
 kind: Ingress
 apiVersion: extensions/v1beta1
 metadata:
-  name: uws-server
-  namespace: uws
+  name: {{ template "uws-api-server.fullname" $ }}-ingress
```

Comment:
Is it appropriate to move this out of the `uws` namespace?

Reply:
It will still be in the uws namespace as set by the ArgoCD (or potentially Helm) config, but this way the namespace is not hard-coded.